litmus-portal/frontend/src/views/CreateWorkflow/CustomWorkflow/TuneWorkflow/index.tsx

```diff
@@ -0,0 +1,399 @@
+        customWorkflows: [
+          ...workflowDetails.customWorkflows,
+          {
+            experiment_name: workflowDetails.customWorkflow.experiment_name,
+            hubName: workflowDetails.customWorkflow.hubName,
+            repoUrl: workflowDetails.customWorkflow.repoUrl,
+            repoBranch: workflowDetails.customWorkflow.repoBranch,
+            yamlLink: workflowDetails.customWorkflow.yamlLink,
+            yaml: YamlString,
+          },
+        ],
+      });
+    } else {
+      experimentArray[
+        workflowDetails.customWorkflow.index as number
+      ].yaml = YamlString;
+      workflowAction.setWorkflowDetails({
+        customWorkflows: [...experimentArray],
+      });
+    }
+  };
+
+  return (
+    <div className={classes.root}>
+      <div className={classes.headerDiv}>
+        <BackButton isDisabled={false} gotoStep={() => gotoStep(0)} />
+        <Typography variant="h3" className={classes.headerText} gutterBottom>
+          {t('customWorkflow.tuneExperiment.headerText')}
+        </Typography>
+        <Typography className={classes.headerDesc}>
+          {t('customWorkflow.tuneExperiment.addKey')}
+        </Typography>
+      </div>
+      <div className={classes.workflowDiv}>
+        <Typography variant="h4">
+          <strong> {t('customWorkflow.tuneExperiment.experimentInfo')}</strong>
+        </Typography>
+        <div>
+          <div className={classes.inputDiv}>
+            <Typography variant="h6" className={classes.mainText}>
+              {t('customWorkflow.tuneExperiment.expName')}:
+            </Typography>
+            <Typography className={classes.mainDetail}>
+              {workflowDetails.customWorkflow.experiment_name}
+            </Typography>
+          </div>
+          <div className={classes.inputDiv}>
+            <Typography variant="h6" className={classes.mainText}>
+              {t('customWorkflow.tuneExperiment.description')}:
+            </Typography>
+            <Typography className={classes.mainDetail}>
+              {getWorkflowDesc()}
+            </Typography>
+          </div>
+          <hr className={classes.horizontalLineHeader} />
+          <div className={classes.inputDiv}>
+            <Typography variant="h6" className={classes.mainText}>
+              {t('customWorkflow.tuneExperiment.sequence')}:
+            </Typography>
+            <Typography className={classes.mainDetail}>
+              {experimentSequence()}
+            </Typography>
+          </div>
+        </div>
+        <hr className={classes.horizontalLineHeader} />
+        {loadingEnv ? (
+          <Loader />
+        ) : (
+          <div className={classes.appInfoMainDiv}>
+            <Typography className={classes.appInfoHeader}>
+              {t('customWorkflow.tuneExperiment.appInfo')}
+            </Typography>
+            <div className={classes.appInfoDiv}>
+              <Typography className={classes.appInfoText}>appns:</Typography>
```

Comment:
same, If added, adding as <appns: "appns:"> in the yaml will avoid it being read as a key.